tests/features/support/steps.js

```diff
@@ -57,6 +58,38 @@ Then("o processo passou pelo nó {string}", { timeout: 60 * 1000 }, async functi
   return;
 });
 
+Then("the bag of {string} has the property {string}", { timeout: 60 * 1000 }, async function (node, property) {
+  await this.getProcessHistory();
+  const nodeState = this.history.find(state => state.node_id === node);
+  const bagHasProperty = _.has(nodeState.bag, property);
```

Comment:
vc precisa carregar o lodash para isso?
Não bastaria fazer um hasOwnProperty()?
Como vc trata propriedades aninhadas, como the bag of X has the property userData.phone?

Reply:
O motivo de carregar o lodash é justamente a resposta da sua pergunta.. o método "has" do lodash trata as propriedades aninhadas, como "userData.phone", "activities[0].data.token", etc..